src/client-side-encryption/client_encryption.ts

```diff
@@ -909,19 +931,6 @@ export interface ClientEncryptionCreateDataKeyProviderOptions {
   keyMaterial?: Buffer | Binary;
 }
 
-/**
```

Comment:
Seems like we had a duplicate `ClientEncryptionRewrapManyDataKeyProviderOptions` definition.